.github/workflows/test-report.yml

```
@@ -0,0 +1,400 @@
+          
+          echo "📄 Verbose file content preview:"
+          head -20 test-verbose.txt 2>/dev/null || echo "No verbose file"
+        continue-on-error: true
+        
+      - name: Parse test results and generate report
+        id: test_results
+        run: |
+          node -e "
+            const fs = require('fs');
+            const path = require('path');
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)